app/src/main/java/np/com/naxa/factsnepal/common/Constant.java

```diff
@@ -1,12 +1,17 @@
 package np.com.naxa.factsnepal.common;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class Constant {
     public static final String KEY_OBJECT = "object";
     public static final String KEY_EXTRA_OBJECT = "extra_object";
     public static final String EXTRA_IMAGE = "extra_image";
     public static String RANDOM_PHOTO = "https://picsum.photos/1000/600/?random";
     public static String FACT_PHOTO = "https://scontent.fktm8-1.fna.fbcdn.net/v/t1.0-9/52638797_2396505700373002_4565827759254798336_n.png?_nc_cat=105&_nc_ht=scontent.fktm8-1.fna&oh=7bbb8ead8c153edfe8e2f1af17f8eef2&oe=5D134EF8";
 
+    public static List<Integer> generatedViewIdsList = new ArrayList<Integer>();
```

Comment:
forgetting to destroy `generatedViewIdsList` will cause uneesary ids to be saved when multiple forms are filled in one session